synapse/config/sso.py

```diff
@@ -36,13 +36,27 @@ class SsoAttributeRequirement:
     """Object describing a single requirement for SSO attributes."""
 
     attribute: str
-    # If a value is not given, than the attribute must simply exist.
-    value: Optional[str]
+    # If a value or values are not given, than the attribute must simply exist.
```

Comment:
according to the schema, one or both of `value` or `values` is required - so I don't understand this comment?